pkg/oauth/server.go

```diff
@@ -223,18 +223,10 @@ func (s *server) RegisterRoutes(server *web.Server) {
 
 	csrfMiddleware := middleware.CSRF("_csrf", "/", s.config.CSRFAuthKey)
 
-	api := root.Group("/api", csrfMiddleware)
-	api.POST("/auth/login", s.Login)
-	api.POST("/auth/logout", s.Logout, s.requireLogin)
-	api.GET("/me", s.CurrentUser, s.requireLogin)
-
 	page := root.Group("", csrfMiddleware)
-	page.GET("/login", webui.Template.Handler, s.redirectToNext)
 	page.GET("/logout", s.ClientLogout)
 	page.GET("/authorize", s.Authorize(webui.Template.Handler), s.redirectToLogin)
 	page.POST("/authorize", s.Authorize(webui.Template.Handler), s.redirectToLogin)
-	page.GET("/", webui.Template.Handler, s.redirectToLogin)
-	page.GET("/*", webui.Template.Handler)
```

Comment:
I think we (used to) send emails linking to `/oauth`, so if we remove this, we're breaking those links, so let's keep the redirect

Reply:
But the route will be handled by the account server. Am I missing something here?

Reply:
Okay, then it's fine. I thought the account server was mounted somewhere else.